src/Workspaces/Core/Portable/CodeFixes/CodeFixContext.cs

```diff
@@ -124,8 +147,34 @@ bool ITypeScriptCodeFixContext.IsBlocking
         {
         }
 
+        /// <summary>
+        /// Creates a code fix context to be passed into <see cref="CodeFixProvider.RegisterCodeFixesAsync(CodeFixContext)"/> method.
+        /// </summary>
+        /// <param name="document">Text document to fix.</param>
+        /// <param name="diagnostic">
+        /// Diagnostic to fix.
+        /// The <see cref="Diagnostic.Id"/> of this diagnostic must be in the set of the <see cref="CodeFixProvider.FixableDiagnosticIds"/> of the associated <see cref="CodeFixProvider"/>.
+        /// </param>
+        /// <param name="registerCodeFix">Delegate to register a <see cref="CodeAction"/> fixing a subset of diagnostics.</param>
+        /// <param name="cancellationToken">Cancellation token.</param>
+        /// <exception cref="ArgumentNullException">Throws this exception if any of the arguments is null.</exception>
+        public CodeFixContext(
```

Comment:
Can we chain the previous constructor to this rather than duplicating the validation?

Reply:
I think you updated the earlier one in the file but not this one?

Reply:
It felt the constructor chain was getting too complex, and we are not really saving much by keeping this couple of lines of duplication.